.gitignore

```diff
@@ -0,0 +1,3 @@
+/vendor/
+composer.json
+composer.lock
```

Comment:
why would you have these files/folder in your project?

Reply:
This was a result of Ash trying to sort out the phpunit - not sure. I can take them out?